samples/blog.yml

```diff
@@ -2,7 +2,7 @@ swagger: '2.0'
 info:
   title: blog
   description: Example for Blog
-  version: 0.1.0
+  version: 9.0.0
```

Comment:
Changing the version in the swagger file here reflects real usage. 

Generated submodules will have the same version as the root module. Since wag is at v9 these should be as well.